test/api/users.coffee

```diff
@@ -0,0 +1,85 @@
+'use strict'
+
+User = require("../../website/src/models/user").model
+app = require("../../website/src/server")
+
+describe "Users", ->
+
+  describe "Deleting Users", ->
+
+    before (done) ->
+      registerNewUser ->
+        done()
+      , true
+
+    it 'Should delete a user', (done) ->
+      async.waterfall [
+        (cb) ->
+          registerManyUsers 1, cb
```

Comment:
Though, doing the look up after the fact would require a second user, as you have done, since you couldn't look up a user using deleted credentials :smile: